apps/ledger-live-desktop/tests/page/abstractClasses.ts

```diff
@@ -26,6 +27,7 @@ export abstract class Component extends PageHolder {
     return await this.page.waitForLoadState("load");
   }
 
+  @step("Wait for page to be fully loaded")
```

Comment:
`wait for network calls to be done` might be more appropriate

Reply:
done